### ISSUE TITLE
Test for Technique G21: Ensuring that users are not trapped in content

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#249]
http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140408/G21
The objective of this technique is to ensure that keyboard users do not become trapped in a subset of the content that can only be exited using a mouse or pointing device. A common example is content rendered by plug-ins. Plug-ins are user agents that render content inside the user agent host window and respond to all user actions that takes place while the plug-in has the focus. If the plug-in does not provide a keyboard mechanism to return focus to the parent window, users who must use the keyboard may become trapped in the plug-in content.
## Procedure
-  Tab through content from start to finish.
- Check to see that keyboard focus is not trapped in any of the content.
- If keyboard focus appears to be trapped in any of the content, check that help information is available explaining how to exit the content and can be accessed via the keyboard.
## QUAIL

The procedure doesn't describe situations where you enter information (autocomplete etc or enter), that will open modals etc. This test seems focused around plugins. It is a general technique, not a HTMLtechnique. The HTML technque for Flash objects is described in the FLASH17 technique

This technique is describing a failure situation and looks like the counterpart of F10.
As such we can make a doable precondition test:
if no objects or applets are used -> F10 pass, G21 pass
if objects or applets are used: F10 cantell, G21 canttell
